Framework/DataHandling/src/LoadRawHelper.cpp

```diff
@@ -1191,47 +1242,31 @@ LoadRawHelper::searchForLogFiles(const std::string &pathToRawFile) {
 
   // File property checks whether the given path exists, just check that is
   // actually a file
-  Poco::File l_path(pathToRawFile);
-  if (l_path.isDirectory()) {
-    g_log.error("In LoadLog: " + pathToRawFile +
-                " must be a filename not a directory.");
-    throw Exception::FileError("Filename is a directory:", pathToRawFile);
+  if (Poco::File(pathToRawFile).isDirectory()) {
+    throw Exception::FileError("Filename is a directory:",
+                               pathToRawFile.toString());
   }
 
   // start the process or populating potential log files into the container:
   // potentialLogFiles
-  std::string l_filenamePart =
-      Poco::Path(l_path.path()).getFileName(); // get filename part only
-  if (isAscii(pathToRawFile) &&
-      l_filenamePart.rfind('_') != std::string::npos) {
+  // have we been given what looks like a log file
+  const auto fileExt = pathToRawFile.getExtension();
```

Comment:
Does this handle case sensitivity? 
AFAIK if a user has a filename like `LOG.TXT` or similar they will no longer load files as this check will fall over leading to an obscure bug in the future I guess

Reply:
The case sensitivity below is less important as were globbing and trying standard naming patterns rather than user input

Reply:
Yes good catch. This would fail on Linux as there is a diversity in cases for these files generally.